frontend/public/components/edit-yaml.jsx

```diff
@@ -515,10 +516,13 @@ export const EditYAML = connect(stateToProps)(
       const klass = classNames('co-file-dropzone-container', {'co-file-dropzone--drop-over': isOver});
 
       const {error, success, stale, yaml, height} = this.state;
-      const {create, obj, download = true, header} = this.props;
+      const {create, obj, download = true, header, kind} = this.props;
       const readOnly = this.props.readOnly || this.state.notAllowed;
       const options = { readOnly, scrollBeyondLastLine: false };
       const model = this.getModel(obj);
+      const sideBar = (create && resourceSidebars.get(kind))
+        ? <ResourceSidebar isCreateMode={create} kindObj={model} height={height} loadSampleYaml={this.loadSampleYaml_} downloadSampleYaml={this.downloadSampleYaml_} />
+        : <ExploreTypeSidebar isCreateMode={create} kindObj={model} height={this.state.height} />;
```

Comment:
We didn't add this to the create forms yet because we wanted it to consistently show up for all types, even those with a samples sidebar. The current design is to have a tabs for the samples and the schema in the sidebar.